mtbf_job_runner.py

```diff
@@ -80,16 +83,23 @@ def add_7mobile_action(self):
         return True
 
     @action(enabled=False)
-    def change_memory(self, memory=0):
+    def change_memory(self):
         #TODO: use native adb/fastboot command to change memory?
         # Make sure it's in fastboot mode, TODO: leverage all fastboot command in one task function
         if self.adb_test():
-            os.system("adb reboot boot-loader")
-        mem_str = str(memory)
-        os.system("fastboot setvar mem " + mem_str)
-        # Preventing from async timing of fastboot
-        time.sleep(5)
-        os.system("fastboot reboot")
+            os.system("adb reboot bootloader")
+            memory = 319
```

Comment:
This one is a workaround;  action runner currently not able to accept parameters.
Will fix later in combo-runner